tower-http/src/services/fs/serve_dir/tests.rs

```diff
@@ -0,0 +1,637 @@
+use crate::services::{ServeDir, ServeFile};
+use brotli::BrotliDecompress;
+use bytes::Bytes;
+use flate2::bufread::{DeflateDecoder, GzDecoder};
+use http::{header, Method, Response};
+use http::{Request, StatusCode};
+use http_body::Body as HttpBody;
+use hyper::Body;
+use std::io::{self, Read};
+use tower::ServiceExt;
+
+#[tokio::test]
```

Comment:
Unrelated to your changes but I see now that we are missing a test for the security issue we had a while back unless I'm misstaken. Where the user could access arbitrary files outside the directory served like: `/foo/bar/c:/secret.txt`

Reply:
From looking at https://github.com/tower-rs/tower-http/pull/204 it doesn't seem like we ever had a test for this. We probably should.

Reply:
yep

Reply:
I've made an issue for it https://github.com/tower-rs/tower-http/issues/251